Remotion/Core/Reflection/MethodInfoAdapter.cs

```diff
@@ -117,14 +119,14 @@ public ITypeInformation GetOriginalDeclaringType ()
       return AttributeUtility.GetCustomAttributes<T> (_methodInfo, inherited);
     }
 
-    public object Invoke (object instance, object[] parameters)
+    public object Invoke (object? instance, object?[]? parameters)
     {
-      ArgumentUtility.CheckNotNull ("instance", instance);
+      ArgumentUtility.CheckNotNull ("instance", instance!);

```

Comment:
null bug
question is, do we want to add a conditional argument check or let the MethodInfo handle this? need test either way.